src/pages/Landing/Features.tsx

```diff
@@ -0,0 +1,41 @@
+import featureOne from "../../assets/1.png";
+import featureTwo from "../../assets/2.png";
+import featureThree from "../../assets/3.png";
+
+const Features = () => {
+  return (
+    <div>
+      <div className="container flex gap-16 flex-col items-center justify-center py-8">
+        <h2>
+          Gain Valuable Insights into{" "}
+          <span className="text-accent">Your Code</span>
+        </h2>
```

Comment:
can you center the content of this `h2`
![image](https://github.com/catalyst-ci-cd/catalyst-frontend/assets/52636794/e41d20db-b929-48e9-82dd-38947ec36022)